include/boost/geometry/strategies/geographic/segment_below_of_box.hpp

```diff
@@ -0,0 +1,58 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2018 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fisikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_GEOGRAPHIC_SEGMENT_BELOW_OF_BOX_HPP
+#define BOOST_GEOMETRY_STRATEGIES_GEOGRAPHIC_SEGMENT_BELOW_OF_BOX_HPP
+
+#include <boost/geometry/strategies/segment_below_of_box.hpp>
+
+namespace boost { namespace geometry
+{
+
+
+namespace strategy { namespace segment_below_of_box
+{
+
+template <typename LessEqual, typename ReturnType, typename PPStrategy>
+struct geographic
```

Comment:
AFAIU this should not be a separate strategy but an internal detail of other strategy.